run/src/adapt.ts

```diff
@@ -1,15 +1,27 @@
 import {Stream} from 'xstream';
 
+declare var window: any;
+
+function getGlobal(): any {
+  if(typeof window !== 'undefined') {
+    return window;
+  }
+  if(typeof global !== 'undefined') {
+    return global;
+  }
+  return this;
+}
+
 export interface AdaptStream {
   (s: Stream<any>): any;
 }
 
-let adaptStream: AdaptStream = x => x;
+getGlobal().adaptStream = (x => x) as AdaptStream;
```

Comment:
Actually I hadn't realized this before, but since the singleton `adaptStream` is now attached to a shared global, e.g. `window`, we need to be careful with naming (unlike before). I'd recommend something likely to be unique, like `window.Cyclejs.adaptStream` since cycle/run already attaches `window.Cyclejs.sinks` for the purposes of the DevTool.